src/lib/containers/UserCardSmall.tsx

```diff
@@ -23,13 +28,53 @@ export const UserCardSmall: React.FunctionComponent<UserCardSmallProps> = ({
   showCardOnHover = true,
   disableLink,
   link,
+  showAccountLevelIcon = false,
   openLinkInNewTab,
   withAvatar = false,
   avatarSize = 'SMALL',
   imageURL,
   ...rest
 }) => {
+
+  const [userBundle, setUserBundle] = useState<UserBundle | undefined>()
   const target = useRef(null)
+  
+  let icon = <Registered />
```

Comment:
Thought icon would be a state variable.  Why don't we do this more often (just use a local variable that we update in the effect)?

Reply:
Good idea. Created a state variable to store the icon.